jac-uk/admin
# Feat/admin 2396 create scc eligibility download for legal exercises

## What's included?
closes #2396 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Go to the preview link: https://jac-admin-develop--pr2400-feat-admin-2396-crea-rjpjdx45.web.app/exercise/S9V9R15MGRjCDHdcRQRZ/reports/eligibility-issues
- Click  `Download SCC Annex report`  to download the report
- Check contents of following sections are correct
    - Candidates who do not meet the Statutory Eligibility Criteria
    - Candidates recommended to proceed and the reasons why they are considered to meet the ASC - Previous Judicial Experience
    - Candidates recommended to discuss and the reasons why they are considered to potentially meet the ASC - Previous Judicial Experience, necessary skills in some other significant way
    - Candidates recommended to reject and the reasons why they are considered to not meet the ASC - Previous Judicial Experience through a quasi-judicial role or showing the skills in some other significant way  Candidates who will not be able to provide 3 years reasonable service when recommendations are made to the Lord Chancellor

Example exercises:
- [Ryan test 1 (Character issues)](https://jac-admin-develop--pr2400-feat-admin-2396-crea-rjpjdx45.web.app/exercise/IbeEVi4Vtrpv9eVFzhZq/reports/eligibility-issues)
- [Ryan test 2 (Character issues)](https://jac-admin-develop--pr2400-feat-admin-2396-crea-rjpjdx45.web.app/exercise/FpV4enhw0yWFb9htWO7y/reports/eligibility-issues)
- [Ryan test 3 (Character issues)](https://jac-admin-develop--pr2400-feat-admin-2396-crea-rjpjdx45.web.app/exercise/DVGGSXxY0MdhbehA1R98/reports/eligibility-issues)
- [Ryan test 4 (Character issues)](https://jac-admin-develop--pr2400-feat-admin-2396-crea-rjpjdx45.web.app/exercise/XEu94ZOPGM1PRZuEhUdH/reports/eligibility-issues)
- [Ryan test 5 (Character issues)](https://jac-admin-develop--pr2400-feat-admin-2396-crea-rjpjdx45.web.app/exercise/g45SG9vPFlhInL6yDonh/reports/eligibility-issues)

🟢 No risk - this is a self-contained piece of work

## Additional context
Include screen grabs, video demo, notes etc.

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required
- Permissions have been added / updated. Details:

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
